course_info/views.py

```diff
@@ -15,7 +14,7 @@
 
 
 @login_required
-@lti_role_required(const.ADMINISTRATOR)
+@lti_permission_required(settings.PERMISSION_ACCOUNT_ADMIN_TOOLS)
```

Comment:
Per the pow-wow we had, lets leave the role check in there in addition to adding this permission check